.github/workflows/test.yml

```diff
@@ -29,7 +29,8 @@ jobs:
       - name: Update pip and install dev requirements
         run: |
           python -m pip install --upgrade pip
+          pip install hatch
```

Comment:
I would restrict the version of hatch that gets installed at least to the major version you're using now.

```
pip install hatch<2.0.0